apps/oc_chef_wm/src/oc_chef_wm_key_util.erl

```diff
@@ -0,0 +1,67 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 80 -*-
```

Comment:
I think what I was trying to do originally it made since to have a util class, but how the code ended up, that is probably the smartest thing to do.